lib/diplomat/rest_client.rb

```diff
@@ -76,6 +76,7 @@ def parse_body
 
     # Get the key/value(s) from the raw output
     def return_value(nil_values=false)
```

Comment:
@piavlo: By providing the keys param, you're asking for an array of strings (they keys). So if @raw is an array of strings this just returns that.